<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>292</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="PlaceholderText">
      <brush brushstyle="SolidPattern">
       <color alpha="128">
        <red>126</red>
        <green>127</green>
        <blue>129</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="PlaceholderText">
      <brush brushstyle="SolidPattern">
       <color alpha="128">
        <red>126</red>
        <green>127</green>
        <blue>129</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="PlaceholderText">
      <brush brushstyle="SolidPattern">
       <color alpha="128">
        <red>126</red>
        <green>127</green>
        <blue>129</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableView" name="tableView"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="txtFirstname">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="128">
              <red>129</red>
              <green>129</green>
              <blue>129</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="128">
              <red>129</red>
              <green>129</green>
              <blue>129</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="128">
              <red>129</red>
              <green>129</green>
              <blue>129</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="placeholderText">
         <string>Firstname</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtLastname">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="128">
              <red>128</red>
              <green>130</green>
              <blue>130</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="128">
              <red>128</red>
              <green>130</green>
              <blue>130</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="128">
              <red>128</red>
              <green>130</green>
              <blue>130</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="placeholderText">
         <string>Lastname</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtAge">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="128">
              <red>128</red>
              <green>130</green>
              <blue>130</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="128">
              <red>128</red>
              <green>130</green>
              <blue>130</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="128">
              <red>128</red>
              <green>130</green>
              <blue>130</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="placeholderText">
         <string>Age</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtWeight">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="128">
              <red>126</red>
              <green>127</green>
              <blue>129</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="128">
              <red>126</red>
              <green>127</green>
              <blue>129</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="128">
              <red>126</red>
              <green>127</green>
              <blue>129</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="placeholderText">
         <string>Weight</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnAdd">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDelete">
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCopy">
        <property name="text">
         <string>Copy</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnLast">
        <property name="text">
         <string>Last Entry</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>717</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
